simtbx/diffBragg/hopper_utils.py

```diff
@@ -1673,6 +1673,56 @@ def __call__(self, x, *args, **kwargs):
         self.g = g
         return f
 
+    def callback(self, x, verbose=True):
```

Comment:
Was this method needed by you ? I cant remember why I deleted it, but seems its gone from master

Reply:
Thanks for pointing this out. It seemed to have slipped through. I removed it.